File: RIT 9th Annual Competition/Problem 3/main.cpp
void caesar()
{
    int shift;
    std::string text;
    
    std::cout << "Enter shift value (1-25): ";
    std::cin >> shift;
    
    if(shift > 25 || shift < 1)
    {
        std::cout << "\nPlease select a value between 1 and 25";
        return;
    }
    else{
        std::cout << "Enter plain text: ";
        std::cin >> text;
    
        std::string ret = "";
        for(int i = 0; i < text.capacity(); i++)
        {
            ret += text[i] + shift;
        }
        std::cout << "Encrypted text: " << ret;
    }
}

File: RIT 9th Annual Competition/Problem 3/main.py
shiftValue = int(input("Enter the shift value 1-25: "))
unencryptedText = input("Enter the text: ")
alpahbet = ['a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z','a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z','a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z']
endString=[]

for x in range(len(unencryptedText)):
	if unencryptedText[x] != ' ':
		endString.append(alpahbet[alpahbet.index(unencryptedText[x])+shiftValue])

for x in endString:
	print(x, end="")